src/templates/sitegate/inline.html

```diff
@@ -10,11 +10,12 @@ <h2 class="pf-widget-headline"></h2>
             <p name="google-login" hidden></p>
           </div>
           <form>
-            <input class="pf-sitegate-field pf-field-full-width" name="username" type="text">
-            <input  class="pf-sitegate-field pf-field-full-width" name="email" type="email">
-            <input class="pf-sitegate-field pf-field-half-width" name="organization" type="text">
-            <input class="pf-sitegate-field pf-field-half-width" name="title" type="text">
-            <div class="pf-sitegate-clear"></div>
+            <input name="username" type="text">
+            <input name="email" type="email">
+            <input class="pf-field-half-width" name="company" type="text">
```

Comment:
one thing i was thinking about with these half widths is how do we handle the dynamic aspect? meaning we can technically shut off the title and leave the company on. in that case how does the form look? any way to make that better? maybe everything just needs to be full widget or something? not sure of the best way to handle. especially on the larger center modal.